### PR TITLE
Periodical update of containers & manifests

### PR DESCRIPTION
**Pull request by yokaze**
This PR for the periodical update of containers & manifests.
https://github.com/cybozu-private/neco-ops/issues/405

### CODE REVIEW

File: argocd/base/deployment.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ spec:
   template:
     spec:
       containers:
-        - image: quay.io/cybozu/argocd:1.5.6.1
```
Comment (zoetrope) [inline argocd/base/deployment.yaml:9]: It's annoying to change the same tag name in multiple places.
How about using kustomize's imageTags?

File: test/Makefile [hunk 2]
```diff
@@ -24,7 +24,7 @@ PROMTOOL_VERSION := 2.12.0
 TELEPORT_VERSION := 4.2.8
```
Comment (zoetrope) [inline test/Makefile:24]: Doesn't this require an upgrade to 4.2.10?

File: argocd/base/deployment.yaml (hunk 1 above)
Comment (kfyharukz) [reply argocd/base/deployment.yaml:9]: I agree.

File: test/Makefile (hunk 2 above)
Comment (kfyharukz) [reply test/Makefile]: Oh, I'm missing an update.

**Pull request merged**